Class/seedSettings.py

```diff
@@ -871,14 +871,23 @@ def parse_settings_string(self, settings_string: str):
         tooltip="If enabled, World Values will turn blue when all Important Checks in a World are found.",
         randomizable=True
     ),
-
-    Toggle(
+    
+    SingleSelect(
         name=settingkey.REVEAL_REPORTMODE,
```

Comment:
This probably should've been using a different setting key - it's going to cause problems when people update and had auto saved settings that had the value stored as a boolean.

```
  File "C:\Users\paulb\source\repos\KH2FM-Mods-equations19\KH2Randomizer\UI\Submenus\HintsMenu.py", line 17, in __init__
    self.add_option(settingkey.REVEAL_REPORTMODE)
  File "C:\Users\paulb\source\repos\KH2FM-Mods-equations19\KH2Randomizer\UI\Submenus\SubMenu.py", line 98, in add_option
    widget = self.make_combo_box(setting_name)
  File "C:\Users\paulb\source\repos\KH2FM-Mods-equations19\KH2Randomizer\UI\Submenus\SubMenu.py", line 238, in make_combo_box
    combo_box.setCurrentIndex(keys.index(self.settings.get(name)))
ValueError: True is not in list
```